cst438_gradebook/src/App.js

```diff
@@ -25,7 +25,7 @@ function App() {
        <Switch>
```

Comment:
I am getting warning for these two pages of unused components. I believe they can be removed, if unused:

src\App.js
  Line 8:8:   'Button' is defined but never used    no-unused-vars
  Line 9:8:   'Radio' is defined but never used     no-unused-vars
  Line 10:9:  'DataGrid' is defined but never used  no-unused-vars
  Line 11:8:  'Grid' is defined but never used      no-unused-vars

src\components\Assignment.js
  Line 1:17:    'Component' is defined but never used   no-unused-vars
  Line 4:8:     'PropTypes' is defined but never used   no-unused-vars
  Line 7:8:     'AppBar' is defined but never used      no-unused-vars
  Line 8:8:     'Toolbar' is defined but never used     no-unused-vars
  Line 9:8:     'Typography' is defined but never used  no-unused-vars